app/components/agenda/agendaitem/agendaitem-controls.js

```diff
@@ -86,19 +96,18 @@ export default class AgendaitemControls extends Component {
   }
 
   @action
-  async postponeAgendaitem(agendaitem) {
-    this.isSavingRetracted = true;
-    agendaitem.set('retracted', true);
-    await agendaitem.save();
-    this.isSavingRetracted = false;
+  async postponeAgendaitem() {
+    this.isSavingDecisionResult = true;
```

Comment:
Convert to a ember-concurrency task and use `isRunning` instead of manually managing this flag? Same comment for `retractAgendaitem` and `resetDecisionResultCode`.